### ISSUE TITLE
add gps coordinates for each commune

### ISSUE THREAD

**Issue by seddik** [othmanus/algeria-cities#32]
This allows users to show statistics on a map and it will help a lot.

**Comment by bensialih** [comment]
Hey @seddik 

I have forked the repo, Im just wondering if you or others are able to contribute.
https://github.com/bensialih/algeria-cities

I have set it up with django and because of the large number of communes in Algeria (1049 according to the database entries), it will take me around 52 days to do 20 entries per day.

Your help is greatly appreciated.
Let me know if you require help with the setup.

**Comment by seddik** [comment]
@bensialih  yes now problem I can help, but I prefer pushing results here in this repo, if the owner isn't active we can push on your fork. we can create a gist just to collect the coordinates then we push here when we are done.

how are you getting the coordinates, are you going to google maps one by one, if that's the case, we can assign wilayas and start working.

**Comment by bensialih** [comment]
hi @seddik 

I have already created a fork.
https://github.com/bensialih/algeria-cities/tree/feature/geolocation/geo
I am hoping to gather the geolocations and then push them onto the master.
If there is a better way of doing this, then let me know.

Speak soon.
Abdel

**Comment by seddik** [comment]
Alright, lets use google maps, let organize things on your repo, you can create something in project, like wilayas and you assign who fills the communes, or create them as issue as you wish.


![image](https://github.com/othmanus/algeria-cities/assets/6674347/d04647db-9d79-4476-bf1e-386c8ad39431)

**Comment by bensialih** [comment]
Hey @seddik,
I am looking at creating this programmatically since there are wilayas we need to do but there is also dairas.
Just writing a script to do this and I will keep you posted.
I will also have to create a video tutorial on how people can create the geolocations.

Keep you posted.
Abdel
